lib/plantuml_helper_patch.rb

```diff
@@ -17,7 +17,8 @@ def heads_for_wiki_formatter
     return if @heads_for_wiki_plantuml_included
     super
     content_for :header_tags do
-      javascript_include_tag('jstoolbar/jstoolbar-textile.min') +
+      javascript_include_tag('jstoolbar/jstoolbar') +
```

Comment:
Are these include tags not stored by the prior invoke of `super`? In my tests it didn't make any difference when I deleted this lines.

Reply:
I'm sorry, but I'm not developer. I just google errors which I got and try fix it. This code work for me on my configuration:

```
Environment:
  Redmine version                4.1.0.stable
  Ruby version                   2.6.5-p114 (2019-10-01) [x86_64-linux]
  Rails version                  5.2.4.1
  Environment                    production
  Database adapter               PostgreSQL
  Mailer queue                   ActiveJob::QueueAdapters::AsyncAdapter
  Mailer delivery                smtp
SCM:
  Subversion                     1.10.4
  Mercurial                      4.8.2
  Bazaar                         2.8.0
  Git                            2.20.1
  Filesystem                     
Redmine plugins:
  plantuml                       0.5.1
```

Reply:
The coincidence that two guys working on the same seemingly abandoned code base at the same time is quite funny. :smile: